docs/DeveloperGuide.adoc

```diff
@@ -690,19 +690,61 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person | store details for contacts
+
+|`* * *` |user |edit person information |update details and fix mistakes
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |undo a command’s effect |fix user mistakes
+
+|`* * *` |user |redo a command’s effect after undoing |reapply the effect of the command if it was not a mistake
+
+|`* * *` |user |add or delete a remark on a person |to store extra information about someone in the addressbook
+
+|`* * *`|user |assign multiple phone numbers and emails to the same person |store both personal and work contact numbers of a person
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *`|user | put persons into groups |manage different social circles
+
+|`* *`|user | view number of edits to the address book during the session | recognize if a mistake has been made during the session
+
+|`* *`|general user | rearrange the order of addressbook entries |have preferred entries on top when using “list” command
+
+|`* *`|general user |get help from within the software |will not have to memorise commands
+
+|`* *`|general user |search for person on social media |find more information on person
+
+|`* *` |user |view typed command history |locate mistakes in typed commands
+
+|`* *` |user |clear typed command history |keep the command list relevant ??
+
+|`* *`| user | shorthand commands | to speed up use of addressbook
+
+|`* *`| user | autocomplete commands | to speed up use of addressbook
+
+|`* *`| any user |login to the correct account |view only the commands available to that account’s permissions
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
 
-{More to be added}
+|`*`|System admin|set expiry date to a person |no need to delete useless contacts periodically
```

Comment:
does this feature only applies to system admin? will normal user wants it too?
Also, where does `system admin` comes in? 

perhaps you should create a user story on why you want to have different type of users